src/adapters/ip_camera/mod.rs

```diff
@@ -159,16 +141,16 @@ impl IPCameraAdapter {
             service: service_id.clone(),
             mechanism: Getter {
                 kind: ChannelKind::Extension {
-                    vendor: Id::new("DLink"),
-                    adapter: Id::new("Adapter"),
-                    kind: Id::new("image_newest"),
-                    typ: Type::Binary,
+                    vendor: Id::new("foxlink@mozilla.com"),
+                    adapter: Id::new("IPCam Adapter"),
+                    kind: Id::new("latest image"),
+                    typ: Type::Unit,
```

Comment:
Ah, np, just noticed we have PR for this already https://github.com/fxbox/foxbox/pull/296 :)